Assets/Scripts/Models/Area/World.cs

```diff
@@ -51,6 +51,11 @@ public World(int width, int height, int depth)
         // Creates an empty world.
         SetupWorld(width, height, depth);
         int seed = UnityEngine.Random.Range(0, int.MaxValue);
```

Comment:
Should this line be removed? Your setting the seed to a random value only to reassign it. (assuming the != Vector3.zero condition is always met.)